.mailmap

```diff
@@ -93,7 +93,8 @@ Olivier Grisel <olivier.grisel@ensta.org> <olivier.grisel@ensta.org>
 Olivier Hervieu <olivier.hervieu@gmail.com> <olivier.hervieu@tinyclues.com>
 Paul Butler <paulgb@gmail.com>
 Peter Prettenhofer <peter.prettenhofer@gmail.com>
-Raghav R V <rvraghav93@gmail.com>
+Raghav RV <rvraghav93@gmail.com>
+Raghav RV <rvraghav93@gmail.com> <ragvrv@gmail.com>
```

Comment:
The first one creates an entry for my correct name mapped to my correct email and the 2nd maps my other e-mail to the first one...